binder/mapping.go

```
@@ -261,10 +262,14 @@
 
 	switch v := any(value).(type) {
 	case string:
-		assignBindData(out, data, key, v, enableSplitting)
+		assignBindData(out, any(data).(map[string][]string), key, v, enableSplitting)
```

golangci-lint (1.62.2)

Error return value is not checked

(errcheck)